### PR TITLE
Reduce memory consumption in ONNXRT adaptor

### PR DESCRIPTION
**Pull request by yuwenzho**
## Type of Change

feature
API not change

## Description

Reduce memory consumption in ONNXRT adaptor:
- Delete unnecessry InferenceSession
- Refine check function for large models 

## How has this PR been tested?

local test

## Dependency Change?

no

### CODE REVIEW

File: neural_compressor/model/onnx_model.py [hunk 1]
```diff
@@ -66,6 +61,23 @@ def __init__(self, model, **kwargs):
         self._get_graph_info()
         self._q_config = None
 
+    def check_large_model(self):
+        """Check model > 2GB."""
+        init_size = 0
+        for init in self._model.graph.initializer:
+            # if size of the initializer > 2GB, return True
+            try:
+                init_bytes = init.SerializeToString()
```
Comment (mengniwang95) [inline neural_compressor/model/onnx_model.py:70]: if the initializer is saved to external file, can you still get its size with this method?

Comment (yuwenzho) [reply neural_compressor/model/onnx_model.py:70]: Yes. `self._model = model if not isinstance(model, str) else onnx.load(model)` will load initializer data

Comment (mengniwang95) [reply neural_compressor/model/onnx_model.py:70]: but if users pass loaded model object to this class, it will not execute onnx.load(model)

Comment (yuwenzho) [reply neural_compressor/model/onnx_model.py:73]: update check of initializer data_location

**Review by hshen14 — APPROVED** [review]

**Review by mengniwang95 — APPROVED** [review]

**Pull request merged**